data/CodingSpecies.json

```diff
@@ -53,4 +53,41 @@
       "description": "Successfully participated in Hacktoberfest 2022"
     }
   ]
```

Comment:
I think you need to update your PR with the changes from `main`, because this PR contains older changes that have been fixed

It is best to use a new branch for each PR that is created from the latest default branch